File: app/src/main/java/com/example/tdm_android/holders/CharacterViewHolder.java
package com.example.tdm_android.holders;

import android.view.View;
import android.widget.TextView;

import androidx.recyclerview.widget.RecyclerView;

import com.example.tdm_android.R;

public class CharacterViewHolder extends RecyclerView.ViewHolder {

    public TextView name;
    public TextView city;

    public CharacterViewHolder(View itemUser) {
        super(itemUser);
        name = itemView.findViewById(R.id.nameTextView);
        city = itemView.findViewById(R.id.cityTextView);
    }

}

File: app/src/main/java/com/example/tdm_android/holders/CharacterViewHolder.kt
package com.example.tdm_android.holders

import androidx.recyclerview.widget.RecyclerView
import android.widget.TextView
import com.example.tdm_android.R
import android.view.View

class CharacterViewHolder(itemUser: View?) : RecyclerView.ViewHolder(
    itemUser!!
) {
    var name: TextView = itemView.findViewById(R.id.nameTextView)
    var culture: TextView = itemView.findViewById(R.id.cultureTextView)

}